spec/fixtures/plugins/remote_plugin.rb

```diff
@@ -0,0 +1,29 @@
+module Fastlane
+  module Actions
+    class RemotePluginAction < Action
+      def self.run(params)
+        UI.success "It works 😇"
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "An example action on how you can import remote code to fastlane"
+      end
+
+      def self.available_options
+        []
+      end
+
+      def self.authors
+        ["KrauseFx"]
+      end
+
+      def self.is_supported?(_platform)
+        true
```

Comment:
It might be a good idea to put a test in the remote plugin example as well, to encourage testing of remote plugins.